controls/cis_al2023.yml

```diff
@@ -0,0 +1,2166 @@
+      - audit_rules_privileged_commands_kmod
+
+  - id: 5.2.3.20
+    title: Ensure the audit configuration is immutable (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - audit_rules_immutable
+
+  - id: 5.2.3.21
+    title: Ensure the running and on disk configuration is the same (Manual)
+    levels:
+      - l2_server
+    status: manual
+
+  - id: 5.2.4.1
+    title: Ensure audit log files are mode 0640 or less permissive (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - file_permissions_var_log_audit
+
+  - id: 5.2.4.2
+    title: Ensure only authorized users own audit log files (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - file_ownership_var_log_audit_stig
+
+  - id: 5.2.4.3
+    title: Ensure only authorized groups are assigned ownership of audit log files (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - file_group_ownership_var_log_audit
+
+  - id: 5.2.4.4
+    title: Ensure the audit log directory is 0750 or more restrictive (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - directory_permissions_var_log_audit
+
+  - id: 5.2.4.5
+    title: Ensure audit configuration files are 640 or more restrictive (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - file_permissions_audit_configuration
+
+  - id: 5.2.4.6
+    title: Ensure audit configuration files are owned by root (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - file_ownership_audit_configuration
+
+  - id: 5.2.4.7
+    title: Ensure audit configuration files belong to group root (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - file_groupownership_audit_configuration
+
+  - id: 5.2.4.8
+    title: Ensure audit tools are 755 or more restrictive (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - file_permissions_audit_binaries
+
+  - id: 5.2.4.9
+    title: Ensure audit tools are owned by root (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - file_ownership_audit_binaries
+
+  - id: 5.2.4.10
+    title: Ensure audit tools belong to group root (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - file_groupownership_audit_binaries
+
+  - id: 5.3
+    title: Ensure logrotate is configured (Manual)
+    levels:
+      - l1_server
+    status: manual
+    related_rules:
+      - ensure_logrotate_activated
+      - package_logrotate_installed
+      - timer_logrotate_enabled
+
+  - id: 6.1.1
+    title: Ensure permissions on /etc/passwd are configured (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - file_groupowner_etc_passwd
+      - file_owner_etc_passwd
+      - file_permissions_etc_passwd
+
+  - id: 6.1.2
```

Comment:
A section is missing. This should be se same as section 6.1.1:
```
  - id: 6.1.2
    title: Ensure permissions on /etc/passwd are configured (Automated)
    levels:
      - l1_server
    status: automated
    rules:
      - file_groupowner_etc_passwd
      - file_owner_etc_passwd
      - file_permissions_etc_passwd
```

Yes, the section `6.1.2` is a duplicated of `6.1.1`. This is a mistake in `CIS_Amazon_Linux_2023_Benchmark_v1.0.0.pdf`, but we  should follow the same section numbers.